common/controllers/framework/framework-step.test.js

```diff
@@ -1,11 +1,14 @@
 const FormWizardController = require('../../../common/controllers/form-wizard')
 const fieldHelpers = require('../../../common/helpers/field')
 const frameworksHelpers = require('../../../common/helpers/frameworks')
+
+const youthRiskAssessmentService = {
+  respond: sinon.stub(),
+}
 const services = {
-  'person-escort-record': require('../../services/person-escort-record'),
-  'youth-risk-assessment': require('../../services/youth-risk-assessment'),
+  personEscortRecord: require('../../services/person-escort-record'),
+  youthRiskAssessment: youthRiskAssessmentService,
```

Comment:
probably want to stub the person escort one here as well

Reply:
I was going to leave this for when the PER service gets converted to a class :)